changeset:   6304:03206d67072c
tag:         tip
user:        Panu Matilainen <pmatilai@redhat.com>
date:        Wed Aug 15 12:07:31 2007 +0300
files:       scripts/find-lang.sh
description:
Make find-lang.sh --with-gnome pick up omf files (rhbz#251400)
Patch from Matthias Clasen.


diff -r 7f45c685a512 -r 03206d67072c scripts/find-lang.sh
--- a/scripts/find-lang.sh	Wed Aug 15 09:12:25 2007 +0200
+++ b/scripts/find-lang.sh	Wed Aug 15 12:07:31 2007 +0300
@@ -106,6 +106,14 @@ s:%lang(C) ::
 s:%lang(C) ::
 /^$/d' >> $MO_NAME
 
+find $TOP_DIR -type f|sed '
+s:'"$TOP_DIR"'::
+'"$NO_ALL_NAME$GNOME"'s:\(.*/omf/'"$NAME"'/'"$NAME"'-\([^/.]\+\).omf\):%lang(\2) \1:
+'"$ALL_NAME$GNOME"'s:\(.*/omf/[a-zA-Z0-9.\_\-]\+/[a-zA-Z0-9.\_\-]\+-\([^/.]\+\).omf\):%lang(\2) \1:
+s:^[^%].*::
+s:%lang(C) ::
+/^$/d' >> $MO_NAME
+
 find $TOP_DIR -type d|sed '
 s:'"$TOP_DIR"'::
 '"$NO_ALL_NAME$KDE"'s:\(.*/doc/kde/HTML/\)\([^/_]\+\)\(.*/'"$NAME"'/\)::

